src/rdo.rs

```diff
@@ -240,10 +243,10 @@ pub fn rdo_mode_decision(
     }
 
     let ref_frame = if luma_mode.is_intra() { INTRA_FRAME } else { LAST_FRAME };
-    let mv = if luma_mode != PredictionMode::NEWMV {
-      MotionVector { row: 0, col: 0 }
-    } else {
-      motion_estimation(fi, fs, bsize, bo, ref_frame)
+    let mv = match luma_mode {
```

Comment:
Once mv is decided here, is "best_mv = mv;" at line 293 still required?

Reply:
Yes, my understanding is that the best_mv needs to be captured in the RDOOutput so that it can be used in the final encode.

Reply:
So, initially  best_mv = MotionVector { row: 0, col: 0 };, then assume intra mode is better than mv modes and no chance for doing "best_mv = mv;", then best_mv is still (0,0). Would this be expected?